### PR TITLE
Stop saving _pods_[field_name] serialized when saving single select pick field values (2.0)

### PR DESCRIPTION
**Pull request by NikV**

### CODE REVIEW

**Comment by NikV** [comment]
3.0 PR on the way as well

**Comment by NikV** [comment]
Fixing issue: https://github.com/pods-framework/pods/issues/2878

**Comment by Shelob9** [comment]
Assigning to @sc0ttkclark for merge.

File: classes/PodsAPI.php [hunk 1]
```diff
@@ -3792,6 +3792,7 @@ public function save_relationships ( $id, $related_ids, $pod, $field ) {
                 $object_type = 'post';
 
             delete_metadata( $object_type, $id, $field[ 'name' ] );
+            delete_metadata( $object_type, $id, '_pods_' . $field[ 'field' ] );
 
             if ( !empty( $related_ids ) ) {
                 update_metadata( $object_type, $id, '_pods_' . $field[ 'name' ], $related_ids );
```
Comment (sc0ttkclark) [inline classes/PodsAPI.php]: `if ( 1 < count( $related_ids ) ) {` should be here

File: classes/PodsAPI.php [hunk 2]
```diff
@@ -3800,7 +3801,7 @@ public function save_relationships ( $id, $related_ids, $pod, $field ) {
                     add_metadata( $object_type, $id, $field[ 'name' ], $related_id );
                 }
             }
-            else
+            if ( 1 < count( $related_ids ) )
```
Comment (sc0ttkclark) [inline classes/PodsAPI.php]: Remove this and the delete_metadata call below since it's added up above

File: classes/PodsAPI.php [hunk 3]
```diff
@@ -3792,16 +3792,18 @@ public function save_relationships ( $id, $related_ids, $pod, $field ) {
...
-
-            if ( !empty( $related_ids ) ) {
-                update_metadata( $object_type, $id, '_pods_' . $field[ 'name' ], $related_ids );
-
-                foreach ( $related_ids as $related_id ) {
-                    add_metadata( $object_type, $id, $field[ 'name' ], $related_id );
-                }
+            delete_metadata( $object_type, $id, '_pods_' . $field[ 'field' ] );
+            
+            if ( 1 < count( $related_ids ) ) {
```
Comment (sc0ttkclark) [inline classes/PodsAPI.php]: Wrap this `if` only around the `update_metadata`, inside the `if ( ! empty( $related_ids ) ) {`

File: classes/PodsAPI.php [hunk 4]
```diff
@@ -3792,16 +3792,18 @@ public function save_relationships ( $id, $related_ids, $pod, $field ) {
...
+
+	            if ( !empty( $related_ids ) ) {
+	                update_metadata( $object_type, $id, '_pods_' . $field[ 'name' ], $related_ids );
```
Comment (sc0ttkclark) [inline classes/PodsAPI.php]: Change this update_metadata function usage to be add_metadata instead

**Pull request merged**